test/acceptance/resources/custom_app.operator.manifest.yaml

```diff
@@ -107,3 +107,14 @@ spec:
     kind: AppConfig
     shortNames:
       - ac
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: admin-to-backend-appconfigs
+  labels:
+    service.binding/controller: "true"
+rules:
+  - apiGroups: ["stable.example.com"]
+    resources: ["appconfigs"]
+    verbs: ["get", "list", "watch", "update", "patch", "delete"]
```

Comment:
do we need `watch`, `patch` and `delete` permissions for tests to pass?

Reply:
This is for CI only and it cleans up the resources after the tests so `delete` is required but `patch` and `watch`  is not required.